<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1512</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lblStatus">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCommit">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Commit settings to device</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Configuration file:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editConfigFile"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnBrowse">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLoadFromFile">
       <property name="text">
        <string>Load settings from selected file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSaveToFile">
       <property name="text">
        <string>Save settings to file...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Basic setup</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="9" column="0" colspan="2">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Expected frequency must be within +/- 10 MHz of actual input frequency</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="editExpectedFreq1">
        <property name="text">
         <string>40</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="lblRefFreq">
        <property name="text">
         <string>TODO!</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Ch 4 expected freq [MHz]</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Ch 3 expected freq [MHz]</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Ch 2 expected freq [MHz]</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Reference frequency [MHz]</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Detected approximate freq [MHz]</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="2">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Leave textbox blank to disable channel</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="editRefFreq">
        <property name="toolTip">
         <string>10-100 MHz</string>
        </property>
        <property name="text">
         <string>25</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="editExpectedFreq3">
        <property name="text">
         <string>1000</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Ch 1 expected freq [MHz]</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="editExpectedFreq2">
        <property name="text">
         <string>40</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="editExpectedFreq4">
        <property name="text">
         <string>1000</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QTabWidget" name="tabWidgetBasic">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="ControllerSettingsWidget" name="tabBasic1">
         <attribute name="title">
          <string>Channel 1</string>
         </attribute>
        </widget>
        <widget class="ControllerSettingsWidget" name="tabBasic2">
         <attribute name="title">
          <string>Channel 2</string>
         </attribute>
        </widget>
        <widget class="ControllerSettingsWidget" name="tabBasic3">
         <attribute name="title">
          <string>Channel 3</string>
         </attribute>
        </widget>
        <widget class="ControllerSettingsWidget" name="tabBasic4">
         <attribute name="title">
          <string>Channel 4</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="lblHardwareType">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Detected Hardware Type: (to be continued!)</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Advanced setup</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QComboBox" name="comboADCclockSource">
        <item>
         <property name="text">
          <string>Phase-locked to external ref</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Phase-locked to internal ref (limited accuracy)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>ADC clock source</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="editOutputDataRate">
        <property name="text">
         <string>100</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="editPFDtargetFreq">
        <property name="text">
         <string>10</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Output data rate [Hz]</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Target PFD freq [MHz]</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="ChannelSettings" name="tab1">
         <attribute name="title">
          <string>Channel 1</string>
         </attribute>
        </widget>
        <widget class="ChannelSettings" name="tab2">
         <attribute name="title">
          <string>Channel 2</string>
         </attribute>
        </widget>
        <widget class="ChannelSettings" name="tab3">
         <attribute name="title">
          <string>Channel 3</string>
         </attribute>
        </widget>
        <widget class="ChannelSettings" name="tab4">
         <attribute name="title">
          <string>Channel 4</string>
         </attribute>
        </widget>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnRunAmplitudeCal">
          <property name="text">
           <string>Run Amplitude Calibration</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnRunSelfTest">
          <property name="text">
           <string>Run Self-Test</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChannelSettings</class>
   <extends>QWidget</extends>
   <header>channel_settings</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ControllerSettingsWidget</class>
   <extends>QWidget</extends>
   <header>controller_settings_widget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
